taker/src/routes.rs

```diff
@@ -64,8 +68,12 @@ pub async fn feed(
     rx_maker_status: &State<watch::Receiver<ConnectionStatus>>,
     rx_maker_identity: &State<watch::Receiver<Option<identify::PeerInfo>>>,
     identity_info: &State<IdentityInfo>,
-    _auth: Authenticated,
+    user: User,
 ) -> EventStream![] {
+    if user.expired {
```

Comment:
❓ What does it mean for the user to be expired? Originally, I thought it was just a way of telling the user that they need to update their password the first time they use the app. If that's the case, why should that block them from getting new events? Having an unsafe, default password doesn't seem to have anything to do with being able to use the UI.

Or are there other ways in which this field could be `true`?

Reply:
I had to come up with an approach to force the user to change the password on first login. The variable `expired` should indicate that the password is _expired_ and needs to be changed. 

> Having an unsafe, default password doesn't seem to have anything to do with being able to use the app.

I think it does 🤷‍♂️ but I'm happy to remove those checks if you think this is better UX.